source/material_model/melt_global.cc

```diff
@@ -213,9 +213,15 @@ namespace aspect
             }
           else
             {
+              // no melting/freezing is used in the model --> set all reactions to zero
```

Comment:
shouldn't this comment go below the next line?

Reply:
Yes, I guess that makes more sense. Fixed in both places.